### ISSUE TITLE
BUG: moving multiple assets into a non-existent destination produces misleading error

### ISSUE THREAD

**Issue by aqw** [psyinfra/onyo#203]
```
❱ onyo -C '/home/aqw/git/inm7/onyo/tests/sandbox/test_main.py/test 2' mv user/* 'user 2'   
ERROR:onyo:
laptop_apple_macbookpro.4 -> user 2 Assets can't be renamed without --rename.
Can't move multiple assets to user 2
laptop_apple_macbookpro.5 -> user 2 Assets can't be renamed without --rename.
Can't move multiple assets to user 2
laptop_apple_macbookpro.6 -> user 2 Assets can't be renamed without --rename.
Can't move multiple assets to user 2
laptop_apple_macbookpro.8 -> user 2 Assets can't be renamed without --rename.
Can't move multiple assets to user 2
laptop_apple_macbookpro.9 -> user 2 Assets can't be renamed without --rename.
No folders or assets moved.
```

**Comment by TobiasKadelka** [comment]
I think it should error when a non-existant destination is given, just the error appearing is wrong?

Also, I think, onyo tries to move to "./user 2", so that creates the message.
Since there is by default no need to specify a destination asset name (just a destination directory, like in the exampe above), but it is still allowed to specify one (when without --rename it must be the same source and destination asset name, but never the less, it is not blocked) this can probably not just be handled with a simple `isdir(destination)` test in the beginning.

**Comment by aqw** [comment]
> I think it should error when a non-existant destination is given, just the error appearing is wrong?

Yes. The error is misleading.

When more than 2 arguments are provided, it should be assumed that the last argument is a directory. If is does not exist, it should error appropriately and not suggest `--rename`.

**Comment by TobiasKadelka** [comment]
Yes, the number of arguments can solve this, thanks!

**Comment by aqw** [comment]
Fixed in 98a7022ed7e0c4e6cde4be0a479c44097c7db13b
